src/main/java/com/google/sps/servlets/Utils.java

```diff
@@ -36,7 +38,7 @@
 import javax.servlet.http.HttpServletRequest;
 
 /**
- * Utility class for OAuth flow helpers
+ * Utility class for common functionality
```

Comment:
This Utils class seems like it will grow very quickly. Are there ways of classifying the functionality that it does, so that it can be broken up into several concepts?

In general, in OOP, more objects/classes aren't bad. There is in fact an argument that every object should have one responsibility: https://en.wikipedia.org/wiki/Single-responsibility_principle#:~:text=The%20single%2Dresponsibility%20principle%20(SRP,the%20class%2C%20module%20or%20function.

Reply:
yes that makes more sense - in the future I think we can split the functionality on a feature by feature basis to organize it better